src/main/java/training_day4/HW/Vehicle.java

```diff
@@ -0,0 +1,76 @@
+package training_day4.HW;
+
+import java.lang.reflect.Field;
+
+public abstract class Vehicle {
+    public final double DEFAULT_FUEL_CONSUMPTION;
+    private double fuelConsumption;
+    private int horsePower;
+    private double fuel;
+
+    public Vehicle(double fuel, int horsePower, double DEFAULT_FUEL_CONSUMPTION){
```

Comment:
Can you explain what is the idea of having two constructors here?

Reply:
In case you want to set a different Default fuel consumption, you use the first constructor. If you instead just want to create a vehicle with the default fuel consumption of 1.25 then you use the second one.